docs/content/developer/advanced/custom-indexer.mdx

```diff
@@ -39,13 +39,13 @@ The most straightforward stream source is to subscribe to a remote store of chec
 
 :::info Unavailable
 
-The IOTA Foundation does not provide this service at the moment.
+The IOTA Foundation does not provide this service at the moment. It's possible to start a fullnode locally and use the REST API to sync.
 
 :::
 
-The checkpoint files are stored in the following format:
+The checkpoints are stored in the following format:
 
-<code>{Networks.iota_move_testnet.indexerRpc+`/<checkpoint_id>.chk`}</code>. You can download the checkpoint file by
+`http://127.0.0.1:9000/api/v1/checkpoints/<checkpoint_id>/full`. You can download the checkpoint by
 sending an HTTP GET request to the relevant URL.
```

Comment:
You can just leave the URL here within block quotes.

Reply:
fixed: https://github.com/iotaledger/iota/pull/6370/commits/949553106df0526dd757e80add65a7f2cb23b17d